### PR TITLE
chore: prepare tokio-util v0.6.1 release

### PR DESCRIPTION
**Pull request by tesaguri**
# 0.6.1 (January 12, 2020)

### Added

- codec: `get_ref()`, `get_mut()`, `get_pin_mut()` and `into_inner()` for
  `Framed`, `FramedRead`, `FramedWrite` and `StreamReader` (#3364).
- codec: `write_buffer()` and `write_buffer_mut()` for `Framed` and
  `FramedWrite` (#3387).

### CODE REVIEW

**Review by Darksonn — APPROVED** [review]

File: tokio-util/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 0.6.1 (January 10, 2020)
```
Comment (Darksonn) [inline tokio-util/CHANGELOG.md:1]:
```suggestion
# 0.6.1 (January 11, 2020)
```

File: tokio-util/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+# 0.6.1 (January 11, 2020)
```
Comment (Darksonn) [inline tokio-util/CHANGELOG.md:1]:
```suggestion
# 0.6.1 (January 12, 2020)
```

**Review by Darksonn — COMMENTED** [review]
Sorry about this :sweat_smile:

**Comment by Darksonn** [comment]
It seems like we have some issues with clippy and tokio-util. We can investigate separately from this PR.

**Comment by taiki-e** [comment]
> It seems like we have some issues with clippy and tokio-util. We can investigate separately from this PR.

clippy issue should probably be fixed by #3413.

**Pull request merged**